src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -11686,14 +11765,14 @@ void Assembler::evex_prefix(bool vex_r, bool vex_b, bool vex_x, bool evex_r, boo
   // instead of not'd
   int byte2 = (vex_r ? VEX_R : 0) | (vex_x ? VEX_X : 0) | (vex_b ? VEX_B : 0) | (evex_r ? EVEX_Rb : 0);
```

Comment:
Comment at [L#11765 
](https://github.com/openjdk/jdk/pull/18476/files#diff-e3576e9c22db89236cdb906f032ff00748ff6d1c21b05277d991d80af75daf3aL11686)
`// P0: byte 2, initialized to RXBR'00mm  => // P0: byte 2, initialized to RXBR'0mmm`

Reply:
Thanks, done.